streams/src/main/java/org/apache/kafka/streams/processor/To.java

```diff
@@ -89,4 +89,11 @@ public int hashCode() {
         throw new UnsupportedOperationException("To is unsafe for use in Hash collections");
     }
 
+    @Override
+    public String toString() {
+        return "To{" +
```

Comment:
nit: could we do a string format for this to read easier?

Reply:
I just didn't bother because there's no place it would actually be printed unless a test is failing. We can give more thought to the string format later on as needed.